resources/web/README.md

```diff
@@ -1,15 +1,85 @@
-
 ### Frontend Web Developer Roadmap
 
 <LevelWithButton :imageRight="false"  desc="A community-created roadmap for modern frontend web development." image="/images/frontend-roadmap.png" button="Check it out!" link="https://roadmap.sh/frontend" />
 
 ### Backend Web Developer Roadmap
 
 <LevelWithButton desc="A community-created roadmap for modern backend web development." image="/images/backend-roadmap.png" button="Check it out!" link="https://roadmap.sh/backend" />
+
+## HTML, CSS & JavaScript Cheat sheets!
+<div class="scrolling-wrapper">
+<div class="scroll-child">
+<iframe src="https://lifeyourway.net/printables/blogging-html-cheat-sheet.pdf" width="550px" height="500px"></iframe>
+</div>
+<div class="scroll-child">
+<iframe src="https://courses.cs.washington.edu/courses/cse154/16sp/cheat-sheets/css-cheat-sheet.pdf" width="580px" height="500px"></iframe>
+</div>
+  <div class="scroll-child">
+   <iframe src="https://cheatography.com/davechild/cheat-sheets/javascript/pdf/" width="600px" height="500px"></iframe>
+  </div>
+</div>
+
+<style scoped>                                                                                                          
+.scrolling-wrapper {                                                                                                      
+display: flex;                                                                                                          
+flex-wrap: nowrap;                                                                                                      
+overflow-x: auto;                                                                                                                                                                                                                             
+}                                                                                                                       
+.scroll-child {                                                                                                             
+flex: 0 0 auto;                                                                                                        
+ margin-right: 36px;                                                                                                     
+font-size: 24px                                                                                                       
+}                                                                                                                     
+</style>                                                                                                                                 
+
+## What Web Development Tools Are There?
+
+### Top 10 Frontend Libraries
```

Comment:
A description of why we'd want to use a framework would be important here before we list out frontend libraries. The audience we're going for here is first and second years.